tests/unit/fem/test_operatorjacobismoother.cpp

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
```

Comment:
How do you run the unit test?

Reply:
`make unittest` (or`make test` for all tests)